### DISCUSSION TITLE
why did you steal a nickname

### DISCUSSION THREAD

**Discussion by n0nexist** [kl3ssydra/kl3ssydra#2] [General]
my friend https://t.me/kl3sshydraa has been in the community since 2019, did u steal the nickname or is it a coincidence? xD

**Comment by kl3ssydra** [comment]
kl3ssydra != kl3sshydraa

Il Lun 15 Mag 2023, 17:46 nønexist ***@***.***> ha scritto:

**Reply by n0nexist** [reply]
that makes sense
